server/db/mongodb/adapter.go

```diff
@@ -1330,8 +1427,8 @@ func (a *adapter) TopicDelete(topic string, hard bool) error {
 	} else {
 		now := t.TimeNow()
 		_, err = a.db.Collection("topics").UpdateOne(a.ctx, filter, b.M{"$set": b.M{
-			"updatedat": now,
-			"deletedat": now,
+			"state":   t.StateDeleted,
```

Comment:
Do you want to update `updatedat` here as well?

Reply:
https://github.com/tinode/chat/blob/1c74854863439fe5defa6d0c099be0d29cd4bf11/server/db/rethinkdb/adapter.go#L1244-L1249

`updatedat` is not updating in rethinkdb

Reply:
Let me check if it needs to be changed too. Maybe I made a mistake. I'll get back soon.

Reply:
Current code is correct. `updatedat` should NOT be updated on state change.